packages/gatsby-theme-ghost-tags/README.md

```diff
@@ -0,0 +1,41 @@
+# gatsby-theme-ghost-members
```

Comment:
Please rename to gatsby-theme-ghost-tags

Reply:
good catch - I thought I had found them all in my cloning